task/hadolint/0.1/hadolint.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  annotations:
+    tekton.dev/categories: Code Quality
+    tekton.dev/displayName: Hadolint
+    tekton.dev/pipelines.minVersion: 0.12.1
+    tekton.dev/platforms: linux/amd64
+    tekton.dev/tags: 'Kubernetes, Misconfiguration'
+  name: hadolint
+  labels:
+    app.kubernetes.io/version: '0.1'
+spec:
+  description: >-
+    This task makes it possible to use Hadolint within Tekton Pipeline.
+
+    A smarter Dockerfile linter that helps you build best practice Docker
+    images. The linter parses the Dockerfile into an AST and performs rules on
+    top of the AST
+  params:
+    - default: ''
+      description: ignore rules.
+      name: ignore-rules
+      type: string
+    - default: './Dockerfile'
+      description: Dockerfile path.
+      name: dockerfile-path
+      type: string
+    - default: tty
+      description: >-
+        The output format for the results [tty | json | checkstyle | codeclimate
+        | gitlab_codeclimate | codacy] (default tty).
+      name: output-format
+      type: string
+  steps:
+    - image: 'ghcr.io/hadolint/hadolint:v2.8.0-alpine'
+      name: lint-dockerfile
+      resources: {}
+      script: |
+        ls -lh
```

Comment:
is that a debug statement?

Reply:
maybe you want to add a "set -e" so if the linter fail the task will fail too,

Reply:
added "set -e" and delete the debug statement "ls -lh"